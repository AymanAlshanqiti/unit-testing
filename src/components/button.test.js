import renderer from 'react-test-renderer';
import Button from '../components/button';
describe('button', () => {
	it('matches snapshot', () => {
		const tree = renderer.create(<Button label="Naif" />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
