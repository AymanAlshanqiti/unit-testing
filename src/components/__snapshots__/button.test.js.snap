// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button matches snapshot 1`] = `
<div
  className="button-style"
>
  Hello 
  Naif
</div>
`;
